staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go

```diff
@@ -183,11 +183,40 @@ func (az *Cloud) CreateOrUpdateSecurityGroup(sg network.SecurityGroup) error {
 	return rerr.Error()
 }
 
+func cleanupSubnetInFrontendIPConfigurations(lb *network.LoadBalancer) network.LoadBalancer {
+	if lb.LoadBalancerPropertiesFormat == nil || lb.FrontendIPConfigurations == nil {
```

Comment:
What's the reason for checking if `LoadBalancerPropertiesFormat` is nil? I don't see it used below

Reply:
+1

Reply:
`lb.FrontendIPConfigurations` is actually `lb.LoadBalancerPropertiesFormat.FrontendIPConfigurations `